gui/wxpython/vdigit/wxdisplay.py

```diff
@@ -856,7 +856,7 @@ def GetSelectedVertex(self, pos):
 
         if not self._validLine(line):
             return -1
-        ftype = Vect_read_line(self.poMapInfo, self.poPoints, self.poCats, line)
+        _ = Vect_read_line(poMapInfo, self.poPoints, self.poCats, lineNearest)
```

Comment:
Same thing here, I suspect that assigning to underscore would cause problems. (Unless you can teach me why it works, I'm open to learn if I'm wrong)